README.md

```diff
@@ -1,2 +1,52 @@
-# task1
-Change
+<h1 align="center">
+  <br>
+  <img src="" width="200">
+  <br>
+  WeatherAPI
+  <br>
+</h1>
+
+<h4 align="center">Simple Weather API built in GO language.</h4>
+
+<p align="center">
+  <a href="#key-features">Key Features</a> •
+  <a href="#how-to-use">How To Use</a> •
+  <a href="#credits">Credits</a>
+</p>
+
+## Key Features
+
+* Simplified Information In JSON format
+    - <img src="" width="200">
+* Without Complex Integration
+    - Input desired town
+* Returned Error
+    - If it can't get information about weather it returns appropriate error
+  
+
+## How To Use
+
+To clone and run this application, you'll need [GIN](https://github.com/gin-gonic/gin) and [GO](https://go.dev/dl/) installed on your computer. From your command line:
+
+```bash
+# Clone this repository
+$ git clone https://github.com/MatijaBregar/WeatherAPI.git
+
+# Go into the repository
+$ cd WeatherAPI
+
+# Run the app
+$ go run main.go
+```
+
+[Optional]
+For better and faster testing you can use [POSTMAN](https://www.postman.com).
+Use localhost:8080/{_city_} for HTTP GET request.
+
+## Credits
+
+This software uses the following technologies:
```

Comment:
It uses many things....from the os/kernel to hardware technologies :D
What would be more exact is "written in go", "using gin framework", "accessing data from the weather api". And since you are mentioning it here, you don't need to do it again up in the readme.